static/main.css

```diff
@@ -1,116 +1,700 @@
+  height: 4rem;
+  color: var(--font-color-light);
+  background: var(--bg-color-dark);
+}
+
+#menu-header {
+  display: flex;
+  align-items: center;
+}
+
+.lavinmq-logo {
+  display: inline-block;
+}
+
+.lavinmq-logo img {
+  display: block;
 }
 
-header a, header a:visited {
-  color: #FFF;
+header a,
+header a:visited {
+  color: #fff;
   text-decoration: none;
 }
 
 header h2 {
   align-self: center;
   margin: 0 0 0 20px;
 }
+/* Navigation end */
 
-main {
-  grid-row: 2 / span 2;
-  grid-column: 2;
-  align-self: start;
-  display: grid;
-  grid-template-columns: repeat(12, 1fr);
-  gap: 20px;
-  width: calc(100% - 40px);
-  justify-self: center;
-  margin-bottom: 20px;
-  position: relative;
+/* Utility classes */
+.text-center {
+  text-align: center;
+}
+
+.text-left {
+  text-align: left;
+}
+
+.font-semibold {
+  font-weight: 600;
+}
+
+.mb-4 {
+  margin-bottom: 1rem;
+}
+
+.mt-4 {
+  margin-top: 1rem;
+}
+
+.ml-0 {
+  margin-left: 0 !important;
+}
+
+.ml-2 {
+  margin-left: 0.5rem;
+}
+
+.pt-4 {
+  padding-top: 1rem;
+}
+
+.pb-6 {
+  padding-bottom: 1.5rem;
+}
+
+.px-6 {
+  padding-left: 1.5rem;
+  padding-right: 1.5rem;
+}
+
+.py-4 {
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+}
+
+.flex {
+  display: flex;
+}
+
+.flex-row {
+  flex-direction: row;
+}
+
+.flex-col {
+  flex-direction: column;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.items-center {
+  align-items: center;
+}
+
+.gap-1	{
+  gap: 0.25rem;
+}
+
+.gap-2 {
+  gap: 0.5rem;
+}
+/* Utility classes end */
+
+/* Buttons */
+.btn-small {
+  font-size: 0.75rem;
+  line-height: 1rem;
+  font-weight: 600;
+  border-radius: 0.5rem;
+  padding: 0.375rem 1rem;
+  border: 1px solid transparent;
+  background: #F7F5F2;
+  color: #141414;
+  box-shadow: none;
+  display: inline-flex !important;
+  flex-direction: row;
+  align-items: center;
+  gap: .5rem;
+  margin-right: 0.5rem;
+
+  &.bg-light-400 {
+    background: #F7F5F2;
+    color: #141414;
+  }
+
+  &:hover {
+    background: #CBCAC7;
+  }
+}
+
+.btn-small-outlined {
+  font-size: 0.75rem;
+  line-height: 1rem;
+  font-weight: 600;
+  border-radius: 0.5rem;
+  padding: 0.375rem 1rem;
+  border: 1px solid rgb(247 245 242 / 0.1);
+  color: #F7F5F2;
+  background: transparent;
+  box-shadow: none;
+  display: inline-flex !important;
+  flex-direction: row;
+  align-items: center;
+  gap: .5rem;
+}
+
+.btn-small-outlined-danger {
+  font-size: 0.75rem;
+  line-height: 1rem;
+  font-weight: 600;
+  border-radius: 0.5rem;
+  padding: 0.375rem 0.5rem;
+  border: 1px solid rgb(247 245 242 / 0.1);
+  color: #F7F5F2;
+  background: transparent;
+  box-shadow: none;
+  display: inline-flex !important;
+  flex-direction: row;
+  align-items: center;
+  gap: .5rem;
+
+  &:after {
+    content: url(img/icon-cross.svg);
+    position: relative;
+    top: 1px;
+  }
+
+  &:hover {
+    background: #2D2C2C;
+  }
+}
+
+.btn {
+  font-size: 0.875rem;
+  line-height: 1.25rem;
+  font-weight: 600;
+  border-radius: 800px;
```

Comment:
800px? shouldn't be 100% or something?

Reply:
Percentage refer to corresponding dimension of the border box so pixels are needed here to be sure to have just rounded corners and not an oval.